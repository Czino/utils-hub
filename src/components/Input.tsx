export const Input = ({ className, ...props }: React.ComponentProps<'input'>) => (
  <input
    className={[
      'w-full text-xs md:text-xl appearance-none bg-transparent text-dark p-2 border border-dark rounded',
      className,
    ].join(' ')}
    {...props}
  />
)
