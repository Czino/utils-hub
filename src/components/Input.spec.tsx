import { render } from '@testing-library/react'
import { Input } from './Input'

describe('Input', () => {
  it('should render Input with styles', () => {
    const { asFragment } = render(<Input className="mt-4" value="text" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
