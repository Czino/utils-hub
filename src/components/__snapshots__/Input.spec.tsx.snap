// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render Input with styles 1`] = `
<DocumentFragment>
  <input
    class="w-full text-xs md:text-xl appearance-none bg-transparent text-dark p-2 border border-dark rounded mt-4"
    value="text"
  />
</DocumentFragment>
`;
